9/problem9.rb

```diff
@@ -0,0 +1,66 @@
+require 'date'
+require 'time_difference'
+
+puts "Enter your Birth date:(DD/MM/YYYY)"
+
+b_date=gets.chomp!
+
+splitted_b_array = b_date.split("/")
+
+bday = Integer(splitted_b_array[0])
+
+bmonth = Integer(splitted_b_array[1])
+
+byear = Integer(splitted_b_array[2])
+
+
+currentDate=DateTime.now
```

Comment:
Try to follow the convention of naming the variables i.e. separate words a variable using underscore like instead of using currentDate, use current_date